pints/_mcmc/_adaptive_covariance.py

```diff
@@ -58,21 +65,6 @@ def ask(self):
         if not self._running:
             self._initialise()
```

Comment:
Can the code from _initialise can't just go in the ask() method, instead of a separate function?

Reply:
Tell has some initialisation code as well

Reply:
Done!